### ISSUE TITLE
My problem

### ISSUE THREAD

**Issue by liuwenhaha** [twni2016/OrganSegRSTN_PyTorch#9]
when I run this 'run.sh' on my  GPU
it will show
'cuda runtime error(2): out of memory '
on my CPU
it will show
'$ Torch: not enough memory: you tried to allocate 0GB'

**Comment by twni2016** [comment]
Is your GPU memory larger than 10G? We cannot run on CPU.

**Comment by liuwenhaha** [comment]
> Is your GPU memory larger than 10G? We cannot run on CPU.

Yes, I'm sure my GPU has 32gb of  memory, and I'm following your run.sh exactly

**Comment by liuwenhaha** [comment]
> Is your GPU memory larger than 10G? We cannot run on CPU.

Since the GPU mode didn't work, I tried the CPU mode, which also didn't work.
Also, I tried to read only fewer files, such as three, which also appeared'cuda runtime error(2): out of memory'
